CHANGELOG.md

```diff
@@ -1,225 +1,258 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
 ### `Added`
-- [#613](https://github.com/SciLifeLab/Sarek/pull/613) - Add Issue Templates (bug report and feature request)
-- [#614](https://github.com/SciLifeLab/Sarek/pull/614) - Add PR Template
-- [#615](https://github.com/SciLifeLab/Sarek/pull/615) - Add presentation
-- [#615](https://github.com/SciLifeLab/Sarek/pull/615) - Update documentation
-- [#620](https://github.com/SciLifeLab/Sarek/pull/620) - Add `tmp/` to `.gitignore`
-- [#625](https://github.com/SciLifeLab/Sarek/pull/625) - Add [`pathfindr`](https://github.com/NBISweden/pathfindr) as a submodule
-- [#639](https://github.com/SciLifeLab/Sarek/pull/639) - Add a complete example analysis to docs
+
+-   [#613](https://github.com/SciLifeLab/Sarek/pull/613) - Add Issue Templates (bug report and feature request)
+-   [#614](https://github.com/SciLifeLab/Sarek/pull/614) - Add PR Template
+-   [#615](https://github.com/SciLifeLab/Sarek/pull/615) - Add presentation
+-   [#615](https://github.com/SciLifeLab/Sarek/pull/615) - Update documentation
+-   [#620](https://github.com/SciLifeLab/Sarek/pull/620) - Add `tmp/` to `.gitignore`
+-   [#625](https://github.com/SciLifeLab/Sarek/pull/625) - Add [`pathfindr`](https://github.com/NBISweden/pathfindr) as a submodule
```

Comment:
You're obsessed with whitespace changes.

Reply:
Just notice there was a beautifier in atom, so run it on CHANGELOG